src/conf_mode/interfaces_ethernet.py

```diff
@@ -428,13 +429,34 @@ def generate(ethernet):
 def apply(ethernet):
     if 'frr_dict' in ethernet and not is_systemd_service_running('vyos-configd.service'):
         FRRender().apply()
-    e = EthernetIf(ethernet['ifname'])
+    ifname = ethernet['ifname']
+    e = EthernetIf(ifname)
     if 'deleted' in ethernet:
         e.remove()
     else:
         e.update(ethernet)
     if 'static_arp' in ethernet:
         call_dependents()
+
+    # If the interface is managed by the VPP DPDK driver, synchronize runtime
+    # parameters between Linux and the corresponding VPP LCP interface
+    if (
+        ethernet.get('vpp', {})
+        .get('settings', {})
+        .get('interface', {})
+        .get(ifname, {})
+        .get('driver')
+        == 'dpdk'
+    ):
```

Comment:
This reads very hard - please change it to:


```suggestion
    if dict_search('vpp.settings.interface.{ifname}.driver', ethernet) == dpdk:
```

Reply:
Thank you for the suggestion. I've made the change.